crates/bevy_internal/Cargo.toml

```diff
@@ -10,24 +10,366 @@ keywords = ["game", "engine", "gamedev", "graphics", "bevy"]
 categories = ["game-engines", "graphics", "gui", "rendering"]
 
 [features]
+default = ["std", "async_executor", "bevy_log"]
+default_no_std = ["edge_executor", "libm", "critical-section"]
+
+# Optional Crates
+
+## Enable animation support, and glTF animation loading
+bevy_animation = ["std", "dep:bevy_animation"]
+
+## Enable asset support
+bevy_asset = ["std", "dep:bevy_asset"]
+
+## Enables Bevy's core rendering pipeline
+bevy_core_pipeline = ["std", "dep:bevy_core_pipeline", "bevy_image"]
+
+# Provides a collection of developer tools
+bevy_dev_tools = ["std", "dep:bevy_dev_tools"]
+
+## Enables gizmo support
+bevy_gizmos = ["std", "dep:bevy_gizmos", "bevy_image"]
+
+## Enables GLTF support
+bevy_gltf = ["std", "dep:bevy_gltf", "bevy_image"]
+
+## Enables image support
+bevy_image = ["std", "dep:bevy_image"]
+
+# Enable input focus subsystem
+bevy_input_focus = ["dep:bevy_input_focus"]
+
+# Enable logging integration
```

Comment:
nit: there is a bit of an inconsistency in this file regarding the level of `#` and e.g. "enable" vs "enables"